android/brave_java_resources.gni

```diff
@@ -710,6 +712,9 @@ brave_java_resources = [
   "java/res/font/poppins_light.ttf",
   "java/res/font/poppins_medium.ttf",
   "java/res/font/poppins_regular.ttf",
+  "java/res/font/roboto_bold.ttf",
```

Comment:
Are these fonts a requirement from the design team?

Reply:
@brave/design-team is this a requirement? Wouldn't it be inconsistent to use this font in just one dialog and different font in other UI elements?

Reply:
@samartnik This should match others. @sujitacharya2005 , can you confirm it is correct fonts.

Reply:
Thanks @samartnik pointing out
In figma it's mentioned Roboto font.
I discussed with @Sam-Sibley regarding this
As per @Sam-Sibley it's used across brave and rewards now it's used poppins font.
We can use these options which is same as Roboto, so no need add these newly added fonts.
android:fontFamily="sans-serif"
android:fontFamily="sans-serif-light"
android:fontFamily="sans-serif-condensed"
android:fontFamily="sans-serif-thin"
android:fontFamily="sans-serif-medium"

So I will remove these newly added fonts.